src/cf.data.pre

```diff
@@ -1261,7 +1261,7 @@ ENDIF
 	  # destination TCP port (or port range) of the request [fast]
 	  #
 	  # Port 0 matches requests that have no explicit and no default destination
-	  # ports (e.g., HTTP requests with URN targets)
+	  # ports
```

Comment:
Would port 0 ACL match an HTTP request with a URN target after this PR changes?

Reply:
No, it would not, because this ACL is ignored:

 > WARNING: foo ACL is used in context without an HTTP request. Assuming mismatch.

Reply:
I am guessing that request parsing fails when the request has an unknown URI scheme. OK. Are there any other known examples of "requests that have no explicit and no default destination ports"? It would be good to leave a hint here if they still exist after we removed the URN use case.